testing/min_init.vim

```diff
@@ -0,0 +1,8 @@
+" This is the minimal init.vim that is needed to run unit tests
+" It basically points to the plenary package so we can kickstart the tests with
+" just a few lines
+set rtp+=.
+set rtp+=../plenary.nvim/
```

Comment:
That seems like a sneaky annoyance ready to catch people who don’t have the same environment. Is this safe?

Reply:
this is meant to be used in CI, not for local development.

this seems to be the way it's done in several repos, namely telescope.nvim, null-ls, nvim-dap-ui & nvim-notify

I can imagine myself running tests manually since I already use plenary.nvim for other stuff. I can update the README of how would one _manually_ run tests.